src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -128,7 +131,36 @@ func importConfig(
 
 	// PUT to admin API
 	result, err := client.PatchClusterConfig(upsert, remove)
+	if he := (*admin.HttpError)(nil); errors.As(err, &he) {
```

Comment:
If you wanted to go really wild you could merge the two conditionals:
```go
	if he := (*admin.HttpError)(nil); errors.As(err, &he) && he.Response.StatusCode == 400
```
but that may be a bit of a long line 😅

Reply:
Yeah, I thought about that and then passed on it -- my eyes kind of glaze over at the third statement in a line :-)